domains/authorization/Authorization.API/API/Models/Organization.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using API.ValueObjects;
+
+namespace API.Models;
+
+public class Organization
+{
+    private Organization(
+        Guid id,
```

Comment:
I am not entirely sure when we are using value objects and not? can you just describe it shortly.

Reply:
Yes. As much as necessary.